v23.1/cdc-queries.md

```diff
@@ -77,18 +87,24 @@ You can use the following functions in CDC transformation queries:
 
 ### Unsupported functions
 
-You can **not** use the following functions with CDC transformations:
+You can **not** use the following functions with CDC queries:
 
 - Functions marked as "Volatile" on the [Functions and Operators page](functions-and-operators.html).
 - Functions listed in the [Limitations](#limitations) section on this page.
 - Functions marked as "Stable" on the [Functions and Operators page](functions-and-operators.html), **except** for those listed previously.
 
 ## Examples
 
-CDC transformations allow you to customize your changefeed for particular scenarios. This section outlines several possible use cases for CDC transformations. 
+CDC queries allow you to customize your changefeed for particular scenarios. This section outlines several possible use cases for CDC queries. 
 
 See [`CREATE CHANGEFEED`](create-changefeed.html) for examples on using the foundational syntax to create a changefeed. For information on sinks, see the [Changefeed Sinks](changefeed-sinks.html) page.
 
+{{site.data.alerts.callout_success}}
+To optimize the `SELECT` query you run in your changefeed statement, use the [`EXPLAIN`](explain.html) statement to view a statement plan. 
+
+Note that `EXPLAIN` does not have access to `cdc_prev`, therefore you will receive an error if your `SELECT` query contains `cdc_prev`.
```

Comment:
optional: we could link this sentence -- somewhere -- to the example here: `### Emit the previous state of a row`

Reply:
Done! Good idea! Add the linkss!